README.md

```diff
@@ -87,22 +106,45 @@ This builds the plugin in `/build/rootfs`
     }
 }
 ```
+Start task:
+```
+$ snapctl task create -t sample-psutil-statistics-task.json
```

Comment:
Please change `sample-psutil-statistics-task.json` to `sample-psutil-tag-task.json`